debian/enterprise-control-center/include/etc/confluent/docker/control-center.properties.template

```diff
@@ -7,6 +7,9 @@ confluent.monitoring.interceptor.topic.replication={{env.get('CONTROL_CENTER_MON
 confluent.controlcenter.internal.topics.replication={{env.get('CONTROL_CENTER_INTERNAL_TOPICS_REPLICATION', env['CONTROL_CENTER_REPLICATION_FACTOR'])}}
 confluent.controlcenter.command.topic.replication={{env.get('CONTROL_CENTER_COMMAND_TOPIC_REPLICATION', env['CONTROL_CENTER_REPLICATION_FACTOR'])}}
 
+confluent.metrics.topic.partitions={{env.get('CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS', env['CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS'])}}
```

Comment:
i'm also not sure that this should be a required property, since its default *should* usually be good